providers/smartlead/objectNames.go

```diff
@@ -0,0 +1,19 @@
+package smartlead
```

Comment:
this file should be in separate PR.

Reply:
Moved here https://github.com/amp-labs/connectors/pull/1006